src/qtism/data/storage/xml/marshalling/SelectionMarshaller.php

```diff
@@ -75,13 +81,18 @@ protected function unmarshall(DOMElement $element)
         if (($value = $this->getDOMElementAttributeAs($element, 'select', 'integer')) !== null) {
             $withReplacement = false;
 
-            if (($withReplacementValue = $this->getDOMElementAttributeAs($element, 'withReplacement', 'boolean')) !== null) {
+            if (($withReplacementValue = $this->getDOMElementAttributeAs(
+                    $element,
+                    'withReplacement',
+                    'boolean'
+                )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place?